internal/rpc/jsonrpc/methods.go

```diff
@@ -18,7 +18,7 @@ import (
 	"strings"
 	"sync"
 	"time"
-
+	"github.com/decred/dcrwallet/wallet/v3/txrules"
```

Comment:
please leave imports sorted alphabetically. gofmt does this already so i'm not sure how this ended up being changed.